lib/Dialect/FIRRTL/Transforms/MergeConnections.cpp

```diff
@@ -212,17 +212,17 @@ bool MergeConnection::peelConnect(StrictConnectOp connect) {
         subConnections[idx].erase();
     }
 
-    return parentType.isa<FVectorType>()
+    return type_isa<FVectorType>(parentType)
                ? builder->createOrFold<VectorCreateOp>(
                      builder->getFusedLoc(locs), parentType, operands)
                : builder->createOrFold<BundleCreateOp>(
                      builder->getFusedLoc(locs), parentType, operands);
   };
 
   Value merged;
-  if (auto bundle = parentType.dyn_cast_or_null<BundleType>())
+  if (auto bundle = firrtl::type_dyn_cast<BundleType>(parentType))
     merged = getMergedValue(bundle);
-  if (auto vector = parentType.dyn_cast_or_null<FVectorType>())
+  if (auto vector = firrtl::type_dyn_cast<FVectorType>(parentType))
```

Comment:
Um these are not  the same thing?! Please ensure this "translation" is not repeated anywhere in this diff/PR.

I was wondering why didn't have the `_or_null` equivalents, looks like we should add them as well after all.